### PR TITLE
Feature/add map

### PR DESCRIPTION
**Pull request by vo333ua**
## JIRA

https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=933&view=planning&selectedIssue=PYTHON-239

## Code reviewers

- [ ] @meelros
- [ ] @BoroviyOrest
- [ ] @Kardan1123
- [ ] @mizin4ik
- [ ] @dimigor
- [ ] @PetrushynskyiOleksii
- [x] @Sasha1152

### Second Level Review

- [ ] @lhalam

## Summary of change

Add map

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: way_to_home/static/src/components/map/map.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+let DefaultIcon = L.icon({
+    iconUrl: icon,
+    shadowUrl: iconShadow
+});
+
+
+export default class MapLeaflet extends Component {
+
+    constructor(props) {
```
Comment (dimigor) [inline way_to_home/static/src/components/map/map.js]: delete constructor, use only `state = {}`

Comment (vo333ua) [reply way_to_home/static/src/components/map/map.js]: done

File: way_to_home/static/src/app.js [hunk 2]
```diff
@@ -3,10 +3,12 @@ import ReactDOM from 'react-dom';
 import Router from 'react-router-dom/BrowserRouter';
 
 import MainRouter from './main_router'
+import Header from "src/components/header/header";
 
 ReactDOM.render(
     <Router>
         <div>
+            <Header/>
```
Comment (Sasha1152) [inline way_to_home/static/src/app.js:11]: This component should be in main_router.js

File: way_to_home/static/src/components/map/map.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.map{
+    width: 100%;
+    height: 100%;
+    zIndex: -1;
+    position: fixed;
+    top: 0;
+    left: 0;
+    margin: 0;
+    padding: 0;
```
Comment (Sasha1152) [inline way_to_home/static/src/components/map/map.css]: for properties margin and padding, zero is value by default

File: way_to_home/static/src/app.js (hunk 2 above)
Comment (vo333ua) [reply way_to_home/static/src/app.js:11]: no, header must be in /home and /settings

File: way_to_home/static/src/components/map/map.css (hunk 3 above)
Comment (vo333ua) [reply way_to_home/static/src/components/map/map.css]: ok

**Review by meelros — DISMISSED** [review]
Everything is good!

**Review by dimigor — DISMISSED** [review]

**Review by meelros — APPROVED** [review]
everything is good!

**Review by dimigor — APPROVED** [review]

**Pull request merged**